lib/wopinator/sanitizer.rb

```diff
@@ -0,0 +1,25 @@
+module Wopinator
+  class Sanitizer
+    class << self
+      def sanitize(s)
+        s.gsub(/[^a-zA-Z0-9\.\-\_\s]/, '_')
+      end
+
+      # RFC 2152 (UTF-7)
+      def decode(s)
+        s.gsub(/\+([A-Za-z0-9\/]*)-?/) { decode_chunk(Regexp.last_match[1]) }
+      end
+
+      def decode_sanitize(s)
```

Comment:
~~I don't see this being used anywhere.~~  Never mind, I see it's being used by wintessa.